articles/sentinel/sentinel-soar-content.md

```diff
@@ -131,13 +173,34 @@ You can find SOAR integrations and their components in the following places:
 | **Microsoft Defender for IoT** | Playbooks | Microsoft | Orchestration and notification |
 | **Microsoft Teams** | [Managed Logic Apps connector](/connectors/teams/)<br><br>Playbooks | Microsoft<br><br>Community | Notifications, <br>Collaboration, <br>create human-involved responses |
 |
+  
+## Minemeld
+
+| Product | Integration components | Supported by | Scenarios |
+| --- | --- | --- | --- |
+| **Minemeld**<br>(Available as solution) | Custom Logic Apps connector<br><br>Playbooks | Microsoft | Create indicator, <br>Enrich incident |
+|
+  
+ ## Neustar IP GEO Point 
+
+| Product | Integration components | Supported by | Scenarios |
+| --- | --- | --- | --- |
+| **Neustar IP GEO Point **<br>(Available as solution) |  Playbooks | Microsoft | Get IP Geo Info |
```

Comment:
```suggestion
| **Neustar IP GEO Point**<br>(Available as solution) |  Playbooks | Microsoft | Get IP Geo Info |
```
Accept this suggestion to remove the space before the trailing asterisks, which negates the boldface formatting.